pyproject.toml

```diff
@@ -27,22 +27,35 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
-    "absl-py",
     "etils[epath]",
     "mujoco",
-    "pytest",
+    "numpy",
     "warp-lang",
 ]
 
+[project.optional-dependencies]
```

Comment:
let's also leave a not or a TODO: that this is temporary and will be undone after we land MJX:Warp

Reply:
done